### PR TITLE
Trip documents refactoring

### PR DESCRIPTION
**Pull request by Polo150202**

### CODE REVIEW

File: app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+                                                              "sans-serif-condensed"))),
+                                              fontWeight = FontWeight.SemiBold)
+
+                                          Button(
+                                              modifier =
+                                                  Modifier.padding(start = 60.dp)
+                                                      .size(width = 100.dp, height = 50.dp),
+                                              onClick = {
+                                                singlePhotoPickerLauncher.launch(
+                                                    PickVisualMediaRequest())
```
Comment (AgeX21) [inline app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt:332]: There is a bug when you try to upload an image then cancel, if you click on accept : with out without name, the app crased.

File: app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+      // make a topBar to switch between shared and personal documents
+      // make a list of documents
+      topBar = {
+        SecondaryTabRow(selectedTabIndex = state) {
+          titles.forEachIndexed { index, title ->
+            Tab(
+                modifier = Modifier.height(70.dp).testTag("tab$title"),
+                text = { Text(title) },
+                selected = state == index,
+                onClick = { state = index })
```
Comment (AgeX21) [inline app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt:158]: Maybe modify the top bar to look like the rest of the app with a blue background and white text (primary and onPrimary)

**Review by AgeX21 — CHANGES_REQUESTED** [review]
LGTM ! Please look at the review i made and adjust the code.

File: app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt (hunk 1 above)
Comment (Polo150202) [reply app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt:336]: I will fix the bug, thanks to point that to me

File: app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt (hunk 2 above)
Comment (AgeX21) [reply app/src/main/java/com/github/se/wanderpals/ui/screens/docs/DocumentsPS.kt]: Well done!

**Review by AgeX21 — APPROVED** [review]
LGTM !

**Pull request merged**